### DISCUSSION TITLE
MetroUI Php

### DISCUSSION THREAD

**Discussion by mesutcakmakci** [olton/Metro-UI-CSS#1863] [Ideas]
Hello, your work fascinated me. I wanted to integrate this with php codes. It will be nice if you take a look. I sent you an invitation

**Comment by nlared** [comment]
https://nf5.nlared.com/

**Reply by mesutcakmakci** [reply]
Thank you for a great work.  Can you share the download link with me? It was very nice work . But the structure my prepared includes all metroUI elements and functions.   To give an example, as follows:
Window Options;
width height close_buton btn_min btn_max window_class caption_class content_class drag  drag_item drag_area shadow  icon_img
title content resize top left win_area close_click min_click max_click double_click drag_start drag_stop drag_on title_dbl_click
resize_on resize_start resize_stop resize_pause win_created win_closed...

$btn[] = [
    'mif' => 'home',
    'class' => 'primary',
    'script' => "alert('bir')",
];
$customButonIdForWindwos=$m->window_custom_buttons($btn); // custom buton auto id return


$win['id']                  = "" . customButonIdForWindwos . "";  
 $win['custom_button']       = "" . customButonIdForWindwos . ""; 
 $win['width']               = '400'; 
 $win['height']              = '400';
 $win['close_buton']         = 'true';
 $win['btn_min']             = 'true';
 $win['btn_max']             = 'true';

$m->window_init($win);//new window create
and ; example documents
flex_container();
flex_center();
img_div_grid($array_img_files);
div_end();
div_end();
Finally, to send front ;
window_exec();

Thus, we can create metro elements that are much simpler and take up less space with just php without writing any html, javascript etc. code on the page.

also inside the class __destruct;
$this->content= ob_get_clean();
can use cache

Of course, MetroUI deserves a better php code than these php codes I wrote.  But this class is helping me get things done right now.
example page
https://webticaret.store/test/
